app/src/main/java/org/apache/taverna/mobile/ui/workflow/WorkflowFragment.java

```diff
@@ -116,6 +123,29 @@ public void onLoadMore(int current_page) {
             }
         });
 
+        mSwipeRefresh.setColorSchemeResources(R.color.colorAccent, R.color.colorAccent, R.color
+                .colorPrimary);
+        mSwipeRefresh.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                if (mConnectionInfo.isConnectingToInternet()) {
+                    if (mSwipeRefresh.isRefreshing()) {
+                        mPageNumber = 1;
+                        mWorkflowPresenter.loadAllWorkflow(mPageNumber);
```

Comment:
set mSwipeRefresh Refreshing true, So that swipe referesh starts the swipe animation